docs/sources/logql/_index.md

```diff
@@ -277,6 +278,58 @@ will get those labels extracted:
 "status" => "200"
 ```
 
+##### pattern
+
+The pattern parser allow to explicitly extract fields from your log lines by defining a pattern expression which matched how the log line is.
+
+For example the following nginx log line:
```

Comment:
Can we specify that this is whitespace-sensitive, and that whitespace is required to distinguish capture groups?

Reply:
it's actually literals that distinguish capture group.